doc/README.md

```diff
@@ -43,6 +43,7 @@ And start using it!
    - [Game Loop](game.md)
    - [Components](components.md)
    - [Audio](audio.md)
+     - [Looping Background Music](bgm.md)
```

Comment:
I think you can't have sub-categories like that, I should have explained better (this being parsed for the html on the flame website is the reason).
You can have
```md
 * Audio
  - [General Audio/or Audio/or something](audio.md)
  - [Looping Background Music](bgm.md)
```

Reply:
fixed like your suggestion https://github.com/flame-engine/flame/pull/163/commits/544fdb72496d3f39470ab1c79c1269a24da750e1